_data/faq.yml

```diff
@@ -2373,107 +2373,6 @@
                 Öffne den Anwendungsmanager deines Gerätes (typischerweise unter Einstellungen → Anwendung → Anwendungsmanager), suche und öffne den Eintrag für c:geo, wähle "Standard wählen" und setze "Unterstützte URLs öffnen" von "Mit dieser App" auf "Immer fragen".\\
                 Nach dieser Änderung sollte dein Internet-Browser wieder auswählbar sein, wenn du "Öffne in Browser" aus c:geo wählst.
 
-- id: calendar-plugin
-  title:
-      en: Calendar plugin
-      de: Kalender-Erweiterung
-  items:
-      - id: plugin-calendar
-        en:
-            title: What does the calendar plugin do?
-            content: |
-                It allows adding event caches to your personal calendar.
-                As it is a plugin it will only work if also c:geo itself is installed on your device.
-                The plugin is only needed for devices running Android 2.3 or 3.x. On all newer Android versions the functions described below do not require the plugin.
-        de:
-            title: Wozu dient die Kalender-Erweiterung?
-            content: |
-                Sie ermöglicht es Event-Caches zu deinen persönlichen Kalender hinzuzufügen.
-                Da es eine Erweiterung ist, funktioniert sie nur wenn auch c:geo auf deinem Gerät installiert ist.
-                Die Erweiterung wird nur für Geräte mit Android-Version 2.3 oder 3.x benötigt. Für alle neueren Android-Versionen wird die Erweiterung zur Nutzung der unten beschriebenen Funktionen nicht benötigt.
-
-      - id: plugin-calendar-howto
-        en:
-            title: How do I add an event to my calendar?
-            content: |
-                When you open an event cache in c:geo select\\
-                Menu → Add to Calendar.
-
-                A new appointment for the event will be created in your calendar.
-        de:
-            title: Wie füge ich ein Event zu meinem Kalender hinzu?
-            content: |
-                Nachdem du einen Event-Cache in c:geo geöffnet hast, wähle:\\
-                Menü → Zu Kalender hinzufügen
-
-                Ein neuer Termin für das Event wird daraufhin in deinem Kalender erstellt.
-
-      - id: plugin-calendar-wrong-time
```

Comment:
I tend to leave the content of this topic available. Our time calculation is still just guessing and users should be aware of that.